src/main/java/org/codeforamerica/messaging/models/Message.java

```diff
@@ -1,32 +1,42 @@
 package org.codeforamerica.messaging.models;
 
+import jakarta.persistence.*;
 import jakarta.validation.constraints.NotBlank;
 import jakarta.validation.constraints.Pattern;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.Value;
-import org.springframework.data.annotation.Id;
-import org.springframework.data.relational.core.mapping.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
 
 import java.time.OffsetDateTime;
 
-@Value
+@Entity
+@Table(name="messages")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 @Builder(toBuilder = true)
-@Table(name = "messages")
 public class Message {
-
     @Id
-    Long id;
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
     @NotBlank
-    @Pattern(regexp = "\\A1?\\d{10}\\z")
-    String to;
+    @Pattern(regexp = "\\A\\+?1?\\d{10}\\z")
```

Comment:
Since you are changing these anyway, would you mind using ^ $ instead? It would make the autogenerated swagger docs slightly better.